models/unit_tests.go

```diff
@@ -68,18 +69,18 @@ func MainTest(m *testing.M, pathToGiteaRoot string) {
 	}
 
 	if err = removeAllWithRetry(setting.RepoRootPath); err != nil {
```

Comment:
the three calls in this file to `removeAllWithRetry` could be replaced with `util.RemoveAll` (or at very least remove loop in `removeAllWithRetry` and have just one call to `util.RemoveAll`), otherwise it'll be 5 * 20 attempts to rm the files

Reply:
done